### PR TITLE
fix AP_begin

### PR DESCRIPTION
**Pull request by AurelienJaquier**
This should avoid indexing out of bounds in AP_begin.

The old AP_begin feature would search for an AP_begin after the last spike. I think it does not make much sense, so I removed this last search.

Hopefully this will fix the segfaults we sometimes get during wheel building & testing.

### CODE REVIEW

**Comment by arnaudon** [comment]
Nice catch! Thanks!

**Review by arnaudon — APPROVED** [review]
This does not change a test, so it means it did not find any after last spike or something in the tests? I'm just curious to see if this will affect features a lot or only in 'edge cases'

File: efel/cppcore/LibV5.cpp [hunk 1]
```diff
@@ -657,7 +657,7 @@ static int __AP_begin_indices(const vector<double>& t, const vector<double>& v,
   minima.push_back(endindex);
   //}
   // printf("Found %d minima\n", minima.size());
-  for (size_t i = 0; i < minima.size() - 1; i++) {
```
[the remark this answers is not in the document]
Comment (AurelienJaquier) [reply efel/cppcore/LibV5.cpp:660]: after this, we look between minima[i] and minima[i+1], and the last item of minima is not the last ahp, but the endindex. So if we stop at the i = last - 1, we still end up looking between last ahp and end index

[the remark this answers is not in the document]
Comment (DrTaDa) [reply efel/cppcore/LibV5.cpp:660]: Ah okok
